mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/services/VirksomhetService.kt

```diff
@@ -77,4 +81,13 @@ class VirksomhetService(
 
     fun hentKontaktpersoner(orgnr: String): List<VirksomhetKontaktperson> =
         virksomhetRepository.getKontaktpersoner(orgnr)
+
+    fun deleteKontaktperson(id: UUID): StatusResponse<Unit> {
+        val (gjennomforinger, avtaler) = virksomhetRepository.koblingerTilKontaktperson(id)
+        if (gjennomforinger.isNotEmpty() || avtaler.isNotEmpty()) {
+            return Either.Left(BadRequest("Kontaktpersonen er i bruk."))
```

Comment:
Skulle vi logget her hvilke id'er kontaktpersonen er i bruk for? Om ikke for brukeren så for vår egen del når en bruker av admin-flate prøver å slette, får beskjed om at det ikke går og så kontakter de oss. Da kan vi relativt raskt sjekke hvilke steder kontaktpersonen er i bruk hos?

Reply:
Den høres ut som en god ide :)